Gemfile.lock

```diff
@@ -17,47 +22,48 @@ GEM
     ast (2.4.2)
     atomos (0.1.3)
     aws-eventstream (1.2.0)
-    aws-partitions (1.655.0)
-    aws-sdk-core (3.166.0)
+    aws-partitions (1.842.0)
+    aws-sdk-core (3.185.1)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.651.0)
       aws-sigv4 (~> 1.5)
       jmespath (~> 1, >= 1.6.1)
-    aws-sdk-kms (1.59.0)
-      aws-sdk-core (~> 3, >= 3.165.0)
+    aws-sdk-kms (1.72.0)
+      aws-sdk-core (~> 3, >= 3.184.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.117.1)
-      aws-sdk-core (~> 3, >= 3.165.0)
+    aws-sdk-s3 (1.136.0)
+      aws-sdk-core (~> 3, >= 3.181.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.4)
-    aws-sigv4 (1.5.2)
+      aws-sigv4 (~> 1.6)
+    aws-sigv4 (1.6.1)
       aws-eventstream (~> 1, >= 1.0.2)
     babosa (1.0.4)
-    bigdecimal (1.4.4)
+    base64 (0.1.1)
+    bigdecimal (3.1.4)
     buildkit (1.5.0)
       sawyer (>= 0.6)
     chroma (0.2.0)
     claide (1.1.0)
-    cocoapods (1.11.3)
+    cocoapods (1.14.1)
```

Comment:
This version update makes CI fails because `Podfile.lock` tracks the CocoaPods version:

<img width="1060" alt="image" src="https://github.com/Automattic/simplenote-ios/assets/1218433/a0863cde-983b-4a94-89e6-351a6115b00a">